src/Symfony/Bundle/FrameworkBundle/Resources/config/web.xml

```diff
@@ -88,5 +88,19 @@
         <service id="disallow_search_engine_index_response_listener" class="Symfony\Component\HttpKernel\EventListener\DisallowRobotsIndexingListener">
             <tag name="kernel.event_subscriber" />
         </service>
+
+        <service id="error_controller" class="Symfony\Component\HttpKernel\Controller\ErrorController" public="true">
+            <argument type="service" id="http_kernel" />
+            <argument>%kernel.error_controller%</argument>
+            <argument type="service" id="error_renderer" />
+        </service>
+
+        <service id="exception_listener" class="Symfony\Component\HttpKernel\EventListener\ExceptionListener">
```

Comment:
So this will register two exception listeners. This one and twig.exception_listener. They do not specify a priority, so it's seems arbitrary which one executes first.

Reply:
related to https://github.com/symfony/symfony/pull/33271#discussion_r316328299 too. WIP.

Reply:
see https://github.com/symfony/symfony/pull/33271#issuecomment-524547139